src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -130,6 +130,7 @@ def path(self, args, kwargs):
             raise CLIError('cloud name unexpectedly empty')
         copy_kwargs = kwargs.copy()
         copy_kwargs.pop('self', None)
+        copy_kwargs.pop('content_type', None)
```

Comment:
Please leave a comment here.

Reply:
This is for the signature change of Python SDK methods (https://github.com/Azure/azure-sdk-for-python/pull/24162):

https://github.com/Azure/azure-sdk-for-python/blob/83ff40e58aad2fbd3f912e2f302a342f1e89a1f9/sdk/network/azure-mgmt-network/azure/mgmt/network/v2017_10_01/operations/_operations.py#L22890-L22899

```py
    @distributed_trace
    def begin_create_or_update(
        self,
        resource_group_name: str,
        virtual_network_name: str,
        parameters: "_models.VirtualNetwork",
        *,
        content_type: Optional[str] = "application/json",
        **kwargs: Any
    ) -> LROPoller["_models.VirtualNetwork"]:
```